ansible/roles/ocp4-workload-quarkus-workshop/tasks/post_workload.yml

```diff
@@ -9,7 +9,7 @@
     data:
       msg: |
         Username: {{ item }}
-        Workshop User Guide: http://web-guides.{{ route_subdomain }}
+        Workshop User Guide: http://web-{{modules}}-guides.{{ route_subdomain }}
```

Comment:
It looks like `modules` is a list?
```
- name: create module list
  set_fact:
    modules: "{{ module_type.split(';') | map('trim') | list }}"
```

Based on that I think we may need something like:
```
- name: lab data for user info
  agnosticd_user_info:
    user: "{{ item }}"
      data:
        msg: |
          Username: {{ item }}
          {% for m in modules %}
          Workshop User Guide: http://web-{{ m }}-guides.{{ route_subdomain }}
          {% endfor %}
```